module/default.nix

```diff
@@ -104,6 +104,16 @@ let
         default = [ ];
         description = "Symbolically link decrypted file to absolute paths";
       };
+
+      copies = mkOption {
```

Comment:
rename option to `cpOnService`. Eventually should add another option to `cpOnActivation`.